packages/serverpod/lib/src/server/websocket_request_handlers/method_websocket_request_handler.dart

```diff
@@ -3,15 +3,13 @@ import 'dart:convert';
 import 'dart:io';
 
 import 'package:meta/meta.dart';
-import 'package:serverpod/protocol.dart';
 import 'package:serverpod/serverpod.dart';
-import 'package:serverpod_shared/serverpod_shared.dart';
 
 /// This class is used by the [Server] to handle incoming websocket requests
 /// to a method. It is not intended to be used directly by the user.
 @internal
 class MethodWebsocketRequestHandler {
-  final _MethodStreamManager _methodStreamManager = _MethodStreamManager();
+  late final MethodStreamManager _methodStreamManager;
```

Comment:
That's a fair concern, I'll have a look 👍

Reply:
I experimented with something like the below, but that will not be very nice where it's used because that code also uses the `Endpoint` version of this. 
```dart
// Experimentation
 MethodWebsocketRequestHandler(
    webSocket: webSocket,
    server: server,
    onClosed: onClosed,
  ).listen(),
  
// But it doesn't harmonize well with the `Endpoint` version of this 
// in server.dart
} else if (uri.path == '/websocket') {
  await _dispatchWebSocketUpgradeRequest(
    request,
    EndpointWebsocketRequestHandler.handleWebsocket,
  );
  return;
} else if (uri.path == '/v1/websocket') {
  await _dispatchWebSocketUpgradeRequest(
    request,
    (
      Server server,
      WebSocket webSocket,
      HttpRequest request,
      void Function() onClosed,
    ) =>
        MethodWebsocketRequestHandler(
      webSocket: webSocket,
      server: server,
      onClosed: onClosed,
    ).listen(),
  );
```
So to not turn this into a major refactor I simply made everything static instead: 0de98358